### PR TITLE
Remove verify after signing when compiling without openssl

### PR DESCRIPTION
**Pull request by gpeacock**
This allows for a sign only build without openssl. It removes the Bogus verifier in that workflow.
Also removes the default Signer traits implementation for send_timestamp_request on wasm builds.

### CODE REVIEW

File: sdk/src/store.rs [hunk 1]
```diff
@@ -418,70 +402,6 @@ impl Store {
         }
     }
 
-    /// Sign the claim and return signature.
```
Comment (mauricefisher64) [inline sdk/src/store.rs:421]: There was actually a reason this was in store.rs and not claim.rs.  claims don't sign themselves.  The are signed by outside actors.  They do not have the global knowledge that a store does.

File: sdk/src/cose_sign.rs [hunk 2]
```diff
@@ -56,33 +54,11 @@ use crate::{
     box_size: usize
 ))]
 pub fn sign_claim(claim_bytes: &[u8], signer: &dyn Signer, box_size: usize) -> Result<Vec<u8>> {
-    // Must be a valid claim.
```
Comment (mauricefisher64) [inline sdk/src/cose_sign.rs:59]: This is the same refactoring that MS made in their PR.  Good simplification.

File: sdk/src/store.rs (hunk 1 above)
Comment (gpeacock) [reply sdk/src/store.rs]: I've reverted all the reworking of signing

**Review by mauricefisher64 — APPROVED** [review]
👍

**Pull request merged**